examples/basic/basic.py

```diff
@@ -1,20 +1,21 @@
 #!/bin/python3
 
-from simpleline.base import App, UIScreen
+from simpleline.render.ui_screen import UIScreen
+from simpleline.base import App
 
-
-class MyApp(App):
-    def application_quit_cb(self):
-        """Show message to user when quitting application"""
-        print("Application is closing. Bye!")
+from simpleline.render.widgets import TextWidget
 
 
 class HelloWorld(UIScreen):
-    title = u"Hello World" # title is printed if there is nothing else
+    title = u"Hello World"  # title is printed if there is nothing else
+
+    def refresh(self, args=None):
+        self._window = [TextWidget("Hello World")]
```

Comment:
I agree, however, I don't want this change it in this PR. There will be follow-up later.